app/cli_commands/__init__.py

```diff
@@ -0,0 +1,12 @@
+import click
+
+from app import app, db
+from app.models.user import User
+
+# https://flask.palletsprojects.com/en/2.0.x/cli/#custom-commands
+
+@app.cli.command("add-user")
+@click.argument("name")
+def add_dummy_users(name):
```

Comment:
This method could be named `add_dummy_user` and be called by another function (like `add_dummy_users`) multiple times, so that developers don't have to create multiple test users by hand, but have them at once like earlier.